resources/views/components/hero/text-overlay.blade.php

```diff
@@ -4,13 +4,23 @@
 
 <div class="hero__wrapper w-full relative">
     <div class="hero__primary-image h-hero max-h-hero w-full bg-cover bg-center relative{{ $loop->first !== true ? ' lazy' : '' }}" @if($loop->first === true) style="background-image: url('{{ $hero['relative_url'] }}')" @else data-src="{{ $hero['relative_url'] }}"@endif></div>
-    <div class="hero__content-position relative md:absolute print:relative md:bottom-0 md:inset-x-0 md:text-white md:white-links md:bg-gradient-darkest">
+    <div class="hero__content-position relative lg:absolute print:relative p-6 lg:p-0 lg:bottom-0 lg:inset-x-0 lg:bg-gradient-darkest lg:pt-20">
         <div class="row">
-            <div class="hero__content relative p-4 pb-0 md:pb-2 md:pt-8 md:pt-20">
-                <div class="hero__title md:drop-shadow-px leading-tight text-2xl mb-1 xl:text-5xl">{!! strip_tags($hero['title'], ['em', 'strong']) !!}</div>
-                @if(!empty($hero['description']))<div class="hero__description md:text-lg content">{!! $hero['description'] !!}</div>@endif
+            <{{ !empty($hero['link']) ? 'a href='.$hero['link'] : 'div' }} class="hero__content relative block {{ !empty($hero['link']) ? 'group no-underline ' : '' }} p-4 lg:pb-2 pl-6 lg:pl-4 pb-8 border-l-12 border-gold border-solid lg:border-0 text-green-900 lg:text-white lg:white-links">
+                <div class="hero__title lg:drop-shadow-px leading-tight mt-4 text-3xl mb-3 lg:text-5xl group-hover:underline">
+                    {!! strip_tags($hero['title'], ['em', 'strong']) !!}
+                </div>
+                @if(!empty($hero['description']))
+                    <div class="hero__description text-lg content">
+                        @if(!empty($hero['link']))
+                            {!! preg_replace(array('"<a href(.*?)>"', '"</a>"'), array('',''), $hero['description']) !!}
+                        @else
+                            {!! $hero['description'] !!}
+                        @endif
+                    </div>
+                @endif
                 @yield('hero-buttons')
-            </div>
+            <{{ !empty($hero['link']) ? '/a' : '/div' }}>
```

Comment:
Feel like the tradeoff of the performance of this isn't worth the inconsistency with splitting this tag up.

Could we stick to full tags here?
`{{ !empty($hero['link']) ? '</a>' : '</div>' }}`